moveit_experimental/moveit_jog_arm/src/jog_interface_base.cpp

```diff
@@ -105,6 +104,15 @@ bool JogInterfaceBase::readParameters(ros::NodeHandle& n)
   error += !rosparam_shortcuts::get("", n, parameter_ns + "/publish_joint_accelerations",
                                     ros_parameters_.publish_joint_accelerations);
 
+  // This parameter name was changed recently.
+  // Try retrieving from the correct name. If it fails, then try the deprecated name.
```

Comment:
Add a TODO to replace this with rosparam_shortcuts::get in the noetic release.

Reply:
done